1.7/administration/installing/cloud/aws/advanced-templates.md

```diff
@@ -0,0 +1,245 @@
+---
+post_title: Advanced AWS Templates
+menu_order: 1
+---
+
+The advanced templates for AWS CloudFormation bring power and flexibility to creating and extending DC/OS clusters. With these advanced templates, you can:
+ 
+ - Instantiate a complete DC/OS cluster on an existing VPC/Subnet combination
+ - Extend and update an existing DC/OS clusters by adding more agents 
+ 
+The advanced templates are used together in conjunction to create a DC/OS cluster. 
+ 
+* **zen.json** This template file orchestrates the component stacks to form an entire DC/OS cluster. Since this is a meta template, it requires the least number of parameters to get a cluster up and running. 
+* **infra.json** This template file defines and creates a DC/OS specific infrastructure that works well with a VPC already created. This is the lowest building block of a DC/OS cluster and the components created in this stack are consumed by the dependent templates (`advanced-master.json`, `advanced-pub-agent.json`, and `advanced-priv-agent.json`).
+* **advanced-master.json** This template file configures the master nodes.
+* **advanced-priv-agent.json** This template configures the private agent nodes.
+* **advanced-pub-agent.json** This template configures the public agent nodes.
+
+
+<!-- Insert graphic -->
+
+
+# System requirements
+
+### Hardware
+
+An Amazon EC2 <a href="https://aws.amazon.com/ec2/pricing/" target="_blank">m3.xlarge</a> instance. Selecting smaller-sized VMs is not recommended, and selecting fewer VMs will likely cause certain resource-intensive services, such as distributed datastores, to not work properly.
+
+### Software
+
+- An Amazon Web Services account with root [IAM](https://aws.amazon.com/iam/) privileges. Advanced privileges are required to install the advanced templates. Contact your AWS admin for more information.
+- AWS [Command Line Interface](https://aws.amazon.com/cli/)
+- [jquery](https://github.com/stedolan/jq/wiki/Installation)
+
+# Quick Start
+You can quickly get up and running with the DC/OS advanced templates. 
+
+## Create your dependencies
+
+Use the `zen.sh` script to create the template dependencies. These dependencies will be used as input to create your stack in CloudFormation.
+
+1.  Save this script as `zen.sh`                                                                     
+
+    ```bash
+    #!/bin/bash
+    set -o errexit -o nounset -o pipefail
+    
+    if [ -z "${1:-}" ]
+    then
+      echo Usage: $(basename "$0") STACK_TAG
```

Comment:
It would be good to be consistent in terms of using just `STACK_TAG` or just `STACK_NAME`. I lean towards `STACK_NAME`